### DISCUSSION TITLE
Reverse Alias Display name

### DISCUSSION THREAD

**Discussion by trikotret** [simple-login/app#732] [Feature Requests]
Hi
Currently the reverse Alias emails come in like this **John Smith - johnsmith@domain.com**. Is there a way to just display the name when the email comes in like this **John Smith** without long email? Especially after the name is saved in address book.

**Comment by nguyenkims** [comment]
Yes this might make sense as the original sender is already included in a SimpleLogin email header.

**Reply by trikotret** [reply]
I think the latest update did it. You guys are amazing.
